subsys/bluetooth/controller/ll_sw/ull_llcp_phy.c

```diff
@@ -563,6 +563,7 @@ static void lp_pu_st_wait_tx_ack_phy_req(struct ll_conn *conn, struct proc_ctx *
 				conn, pu_select_phy_timing_restrict(conn, ctx->data.pu.tx));
 			ctx->state = LP_PU_STATE_WAIT_RX_PHY_UPDATE_IND;
 			ctx->rx_opcode = PDU_DATA_LLCTRL_TYPE_PHY_UPD_IND;
+			llcp_tx_resume_data(conn, LLCP_TX_QUEUE_PAUSE_DATA_PHY_UPDATE);
```

Comment:
Why doesn't the `llcp_tx_resume_data(conn, LLCP_TX_QUEUE_PAUSE_DATA_PHY_UPDATE);` in line 573 work?

Reply:
That line works for peripheral, but causes problems for Central EBQ conformance tests, and is removed in PR #46394 
Waiting for PR46394 to be approved and merged.

Reply:
Or actually, I'll move the change into PR46394, and close this one